editor/test/effects.js

```diff
@@ -140,4 +147,185 @@ describe( 'effects', () => {
 			} ] ) );
 		} );
 	} );
+	describe( '.REQUEST_POST_UPDATE', () => {
+		describe( 'requestPostUpdate()', () => {
+			it( 'should trigger a success action on success of request', () => {
+				const dispatch = sinon.spy();
+				const action = savePost( 'ID', [ 'edits' ] );
+				const post = 'I is post';
+				const wpApiMockSuccess = {
+					data: post,
+					success: false,
+					Post( args ) {
```

Comment:
In addition to that, I am not sure whether this is a good approach for mocking the `wp.api` stuff.  Any improved ideas would be very welcome!